coins.json

```diff
@@ -9,8 +9,13 @@
         "curve": "secp256k1",
         "publicKeyType": "secp256k1Extended",
         "explorer": "https://explorer.nebulas.io/#/tx/",
-        "url": "https://nebulas.io",
-        "rpcNodeInfo": "https://github.com/nebulasio/go-nebulas"
+        "info": {
+            "url": "https://nebulas.io",
+            "node": "https://github.com/nebulasio/go-nebulas",
+            "publicNode": "https://mainnet.nebulas.io",
```

Comment:
Does everything have a public node?

Reply:
As part of the requirements to provide a public node: https://developer.trustwallet.com/wallet-core/newblockchain#requirements, some might have exceptions as bitcoin and similar chains, most of the other projects provide a public node for anyone to use.

Reply:
👍